avro/src/config.ini

```diff
@@ -0,0 +1,9 @@
+[kafka]
+bootstrap_servers = pkc-12576z.us-west2.gcp.confluent.cloud:9092
+sasl_mechanisms =  PLAIN
+security_protocol = SASL_SSL
+sasl_username = Q33EQBHKL5WXCPKM
+sasl_password = opnZuTlG2DulwL0IvJu/BKobYrVLNO1zP4chLthgYoXZiPPXuxD4nnjQwEA6DIvg
+
+[consumer]
+auto_offset_reset = latest
```

Comment:
Configs looks good to me, do you know what possible values available for auto offset reset ? What is latest means ?

Reply:
yes i know some, as soon as the message is read from the topic it commits an offset( the way i think it as index of an array) to keep track what messages consumer has  read . latest here means it reads only the new message from the the topic whenever we run the consumer and other config option is from beginning it reads all the messages from the beginning in the topic.